cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/RepositoryManager.java

```diff
@@ -363,8 +363,7 @@ public String getFileHash(final Path relativePath, final String commitHash)
 
   @Nullable
   private String getFileHash(final Path relativePath, final RevCommit commit)
-      throws IOException,
-      GitAPIException {
+      throws IOException{
```

Comment:
nit: should be an empty space before `{`

Reply:
fixed